docs/tutorials/control/flutter-app.md

```diff
@@ -69,6 +69,12 @@ For this walkthrough, you only need to install the following:
   - When prompted, do install Cocoapods.
     You need it to support the iOS simulator.
 
+{{% alert title="Flutter version" color="note" %}}
+We recommended using Flutter 3.19.6, as this sample app was tested with this version.
+`fvm` is a useful tool for targeting specific flutter versions.
+You can run `fvm use 3.19.6` to target Flutter 3.19.6.
```

Comment:
As a naive user, I'm not sure when/where I'd run this command. Is it after a certain step in this guide? Is it every time I build or run the app?

Reply:
not every time but you can run it whenever before building the app. I'm honestly not sure if this needs to be clarified for most of our target audience (at this point more experienced developers) but I'll add a sentence